packages/components/TranscriptForm/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import CustomAlert from './CustomAlert';
+
+const TranscriptForm = ({ ...props }) => {
+
+  const [ title, setTitle ] = useState(props.title);
+  const [ description, setDescription ] = useState(props.description);
+  const [ isValidated, setValidationStatus ] = useState(false);
+  const [ formData, setFormData ] = useState({});
+  const [ uploadCompleted, setUploadCompletion ] = useState(props.uploadCompleted);
+  const [ notificationMessage, updateNotificationMessage ] = useState(props.notification);
+
+  useEffect(() => {
+    if (!props.uploadCompleted) {
+      const alert = <CustomAlert
+        show={ true }
+        dismissable={ true }
+        variant={ 'danger' }
+        heading={ 'Error could not contact the server' }
+        message={ <p>There was an error trying to create this transcript on the server</p> }
+      />;
+      updateNotificationMessage(alert);
+    }
+  }, [ uploadCompleted ]);
+
+  const handleFileUpload = e => {
+    const file = e.target.files[0];
+
+    if (!title) {
+      setTitle(file.name);
+    }
+
+    const fileObj = {
+      title: title,
+      description: description,
+      file: file,
+      type: file.type
+    };
+
+    if (file.path) {
+      fileObj.path = file.path;
+    }
+    setFormData(fileObj);
+  };
+
+  const sendRequest = () => {
+    const tmpObj = {
+      ...formData,
+      title: title,
+      description: description
+    };
+
+    setFormData(tmpObj);
+
+    props.handleSaveForm(tmpObj);
+  };
+
+  const handleSubmit = (event) => {
+    const form = event.currentTarget;
+    event.preventDefault();
+    event.stopPropagation();
+
+    if (!form.checkValidity()) {
+      setValidationStatus(true);
+    }
+    else {
+      setValidationStatus(true);
+
```

Comment:
```js
// after event.* function executions
setValidationStatus(true);
// then do the if/else 
if (form.checkValidity()) {
    sendRequest();
}
```

This is how I would refactor it, the only thing that differs between the two if/else is that you send the request.